felles/testutilities/src/main/java/no/nav/vedtak/felles/testutilities/cdi/CdiAwareExtension.java

```diff
@@ -0,0 +1,43 @@
+package no.nav.vedtak.felles.testutilities.cdi;
+
+import java.lang.reflect.Method;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.InvocationInterceptor;
+import org.junit.jupiter.api.extension.ReflectiveInvocationContext;
+import org.junit.jupiter.api.extension.TestInstanceFactory;
+import org.junit.jupiter.api.extension.TestInstanceFactoryContext;
+import org.junit.jupiter.api.extension.TestInstantiationException;
+
+public class CdiAwareExtension implements TestInstanceFactory, InvocationInterceptor {
```

Comment:
Burde vi ikke heller bruke @WeldExtension istedet for å rulle vår egen ?

Reply:
Men uansett en god start

Reply:
👍 fjerner herfra - bruk det biblioteket du vil

Reply:
jeg stoppet i manglende Weld-forståelse. Kom et stykke, men ga opp